tests/test_jx/test_metadata.py

```diff
@@ -121,28 +122,28 @@ def test_get_nested_columns(self):
             "query": {
                 "select": ["name", "table", "type", "nested_path"],
                 "from": "meta.columns",
-                "where": {"term": {"table": table_name}}
+                "where": {"eq": {"table": table_name}}
             },
             "expecting_list": {
                 "meta": {"format": "list"},
                 "data": [
                     {"table": table_name, "name": "_id", "type": "string", "nested_path": "."},
                     {"table": table_name, "name": "_a", "type": "nested", "nested_path": "."},
-                    {"table": table_name, "name": "_a.b", "type": "string", "nested_path": ["_a", "."]},
-                    {"table": table_name, "name": "_a.v", "type": "double", "nested_path": ["_a", "."]},
+                    {"table": table_name + "._a", "name": "_a.b", "type": "string", "nested_path": ["_a", "."]},
+                    {"table": table_name + "._a", "name": "_a.v", "type": "number", "nested_path": ["_a", "."]},
```

Comment:
This change is wrong.  One of the differences between a database and a snowflake is that the snowflake sees every column in a database schema as belonging to every table in the database schema.  Asking for all columns of "testing._a" will return the same set of columns as asking for all columns of "testing"; only the columns names are different.